src/platform/i386/lapic.c

```diff
@@ -220,9 +241,35 @@ lapic_find_localaddr(void *l)
 	lapic = (void *)(addr);
 	printk("\tlapic: %p\n", lapic);
 
+	readmsr(MSR_APIC_BASE, &lo, &hi);
+	assert(lo & (1 << 8)); 	/* assume we are the BSP */
+	/* instead of using bit 11 ("enable"), we use the SIV LAPIC control register */
+
 	return addr;
 }
 
+void
+lapic_init(void)
+{
+	u32_t version;
+
+	assert(lapic);
+	lapic_write_reg(LAPIC_SIV_REG, LAPIC_SIV_ENABLE);
```

Comment:
If I remember correctly, we need to set the vector number for SIV here. 
The vector number for SIV should be of `0xpF` where the lower order 4 bits are all 1 and 0<=p<=F(this is obvious I know) and that the SIV handler should not `ack` the interrupts.

Reply:
I will have a look at this later.

Reply:
This comment was made by @gparmer, so I am not sure about the meaning of this comment here.
Maybe Gabe could explain more about this?